### PR TITLE
Hungarian localisation of core

### PR DESCRIPTION
**Pull request by 2colours**

### CODE REVIEW

File: tools/templates/L10N/HU [hunk 1]
```diff
@@ -0,0 +1,752 @@
+# This file contains that German translation of features of the
```
Comment (MasterDuke17) [inline tools/templates/L10N/HU:1]: s/that German/the Hungarian/ (assuming of course that “German” is not Hungarian for “Hungarian”)

Comment (2colours) [reply tools/templates/L10N/HU:1]: Good catch... most definitely work in progress.

File: tools/templates/L10N/HU [hunk 2]
```diff
@@ -0,0 +1,752 @@
...
+# Raku Programming Language.
+#
+# CONTRIBUTORS: Elizabeth Mattijsen
```
Comment (lizmat) [inline tools/templates/L10N/HU:4]: This also needs updating :-)

**Comment by 2colours** [comment]
Don't worry, I haven't forgotten about it but it's not really ready. This week I'll be probably buried with work to finish before a 1-week holiday, plus actually finishing the conference talks. So please hang on, this might have to wait a bit.

**Comment by 2colours** [comment]
What command `False`˙? :D

**Comment by lizmat** [comment]
/me hopes to be able to mention a usable Hungarian localization in her advent post  *nudge* *nudge*

**Comment by 2colours** [comment]
Sorry I was on a "worcation" in Germany last week. Need to see the priority list now. 😅 Honestly, after this month, it's just nice to find an activity that doesn't cost an egregious amount of money...

**Comment by 2colours** [comment]
Thank you for your patience.

This is "as good as I could", meaning that I omitted some (roughly 10, I think) words from the translations, where I really couldn't think of anything. It's really tough with the snip-snitch-spurt-slurp-etc words that are slangy/idiomatic C2 words in English as well, pretty much.

**Comment by lizmat** [comment]
Thanks!  Going to merge after the release!

**Pull request merged**